src/lib/init.ts

```diff
@@ -48,9 +48,13 @@ function initShopifyInternationalScript({ env, merchantId, shopifyInternational,
     url.searchParams.append("merchant", merchantId)
     url.searchParams.append("market", marketId)
     url.searchParams.append("locale", language.toLowerCase())
-    const attributes = { "nosto-language": language, "nosto-market-id": marketId }
+    const attributes = {
+      ...options?.attributes,
+      "nosto-language": language,
+      "nosto-market-id": marketId
+    }
     const loader = scriptLoader ?? defaultScriptLoader
-    return loader(url.toString(), { attributes })
+    return loader(url.toString(), { ...options, attributes })
```

Comment:
Can we simplify this? 
```js
return loader(url.toString(), { options.position, attributes })
```

or  maybe

```js
const attributes = {
      "nosto-language": language,
      "nosto-market-id": marketId
    }
    const loader = scriptLoader ?? defaultScriptLoader
    return loader(url.toString(), { ...options, attributes })
```

Reply:
The first option is not valid JS code and will also break once more option attributes are added
The second snippet is missing the attributes merge